snapcraft-options/snapcraft-sdk.yaml

```diff
@@ -0,0 +1,76 @@
+name: deliveryoptimization-sdk
+base: core20 # the base snap is the execution environment for this snap
+version: '0.1'
+summary: Ubuntu Core 20.04 DO
+description: |
+  SDK snap to simulate and test the communication between the Delivery Optimization Client Ubuntu Core snap and the ADU Snap.
```

Comment:
update to: "a consuming snap". Presumably any other snap could use the DO snap.

Reply:
Re-worded the comment